utils/fetchLsp8Metadata.ts

```diff
@@ -151,7 +151,9 @@ const getLsp8Metadata = async (
 
     // in order to get full url we combine URI it with tokenId (must be lowercased)
     // we also resolve url as uri might be ipfs link
-    url = resolveUrl(uri + tokenIdParsed(tokenId, tokenIdFormat).toLowerCase())
+    url = resolveUrl(
+      uri.trim() + tokenIdParsed(tokenId, tokenIdFormat).toLowerCase()
```

Comment:
Where is this white space? If it's at the end it will also cause a problem when using fetchData using erc725, hmmmm.

Reply:
It was at the end of the decoded base URI. But yeah they've fixed it so it's no longer needed. Removing.